src/python/pull_gsheets_translations_mvp.py

```diff
@@ -0,0 +1,299 @@
+#####################
+# PULLING CORONARTACKER CONTENT TRANSLATIONS FROM GOOGLE SHEETS TO GITHUB REPOSITORY
+#####################
+
+
+#####################
+# AUTHORS
+#####################
+# @KristianR
+# @NickG
+
+
+#####################
+# PACKAGES
+#####################
+#https://docs.python.org/3/tutorial/modules.html
+from json import dump
+from gspread import authorize
+from google.oauth2.service_account import Credentials
+from pandas import Series, DataFrame
+from os import path, mkdir
+from string import punctuation
+
+from nltk import pos_tag, download
+from nltk.tokenize import word_tokenize
+download('punkt')
+download('stopwords')
+download('averaged_perceptron_tagger')
+
+#####################
+# CONSTANTS
+#####################
+scope = ['https://spreadsheets.google.com/feeds',
+'https://www.googleapis.com/auth/drive']
+
+CREDENTIALS_PATH = '../../.my.goauth'
+TRANSLATION_SHEETS_REGEX = ' - Master Sheet'
+TRANSLATION_SHEETS_REGEX_OLD = "(OLD)"
+OUT_DIR = '../../client/public/locales/'
+#OUT_DIR = 'translations/'
+LANGUAGES_TO_PULL = ['English','Dutch (Netherlands)','Spanish','Italian','French','Russian']
+LANGUAGE_LETTERS_DICT = {
+	'English':'en',
+	'Dutch':'nl',
+	'Spanish':'es',
+	'Italian':'it',
+	'French':'fr',
+	'Russian':'ru'
+}
+
+def convert_to_camelCase(value):
+	""" Converts a string to camelCase and removes punctuation.
+
+	Arguments:
+		value {[string]} -- [string from the 'value' column in language_df]
+
+	Returns:
+		[string] -- [returns a string with no punctuation, in camelCase format.]
+	"""
+	# if value is a string with more than one word
+	if len(value.split()) > 1:
+		#converts value to camelCase
+		camelCase = value.split()[0].lower() + " ".join(value.split()[1:]).title().replace(" ","")
+		#removes punctuation
+		return camelCase.translate(str.maketrans('', '', punctuation))

```

Comment:
For consistency, let's pick one depunctuation method, `translate(str.maketrans('', '', punctuation))` or `depunctuate`, and use it everywhere depunctuation is done